### PR TITLE
Update query to get car efficiency on Battery Health dashboard

### PR DESCRIPTION
**Pull request by jheredianet**
Just a little improve in how to get the car efficiency to have a better y-axis representation on kWh.
And other important improve in query is to avoid NULLs if the car doesn´t have a long charging session on the last month, if so, it will show an explanation on the "Information" icon with details.

bh

### CODE REVIEW

File: grafana/dashboards/battery-health.json [hunk 1]
```diff
@@ -97,7 +97,7 @@
         "type": "postgres",
         "uid": "TeslaMate"
       },
-      "description": "**Usable (now)** is the estimated current battery capacity. \n\n**Usable (new)** is the estimated Battery Capacity since you begun to use Teslamate. That's why, the more data you have logged from your brand new car the better. ",
+      "description": "**Usable (now)** is the estimated current battery capacity.\n\nIf you see just '1.0 kWh' here, it means that you need at least a long charge session on the lasth month.\n\n**Usable (new)** is the estimated Battery Capacity since you begun to use Teslamate. That's why, the more data you have logged from your brand new car the better. ",
```
Comment (Sjorsa) [inline grafana/dashboards/battery-health.json:100]: I think "lasth" should be "last" :P

Comment (jheredianet) [reply grafana/dashboards/battery-health.json]: Great catch! Thank you. 
 It's fixed now.

**Comment by davidmarinangeli** [comment]
When will we see a release with the Battery Health dashboard? :S @jheredianet

**Comment by jheredianet** [comment]
> When will we see a release with the Battery Health dashboard? :S @jheredianet

Yeah, it's up to @adriankumpf 
In the mean time you can use it or other dashboards from my repository at https://github.com/jheredianet/Teslamate-CustomGrafanaDashboards

Cheers!

**Comment by davidmarinangeli** [comment]
@jheredianet cheers man, I'll check them!

**Comment by adriankumpf** [comment]
Thank you!

**Pull request merged**